manager/src/main/java/org/openremote/manager/asset/AssetModelResourceImpl.java

```diff
@@ -73,13 +75,12 @@ public Map<String, MetaItemDescriptor<?>> getMetaItemDescriptors(RequestParams r
     }
 
     @Override
-    public Response getValueDescriptorSchema(RequestParams requestParams, String hash, String name) {
+    public JsonNode getValueDescriptorSchema(RequestParams requestParams, String name, String hash) {
         JsonNode schema = assetModelService.getValueDescriptorSchema(name);
         if (schema == null) {
             LOG.log(Level.INFO, "Could not find value descriptor: '" + name + "'");
```

Comment:
Maybe drop the level of this as I don't know what value it adds cluttering up the manager logs page